packages/libs/react-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,21 +1,52 @@
-import React, { forwardRef } from 'react';
-import { arrowVariants, container } from './Tooltip.css';
-
-export interface ITooltipProps {
-  placement?: 'top' | 'bottom' | 'left' | 'right';
-  children: React.ReactNode;
+import { Box } from '@components/Layout';
+import { mergeRefs } from '@react-aria/utils';
+import type { FC, ReactElement, ReactNode } from 'react';
+import React, { cloneElement, useRef } from 'react';
+import { useTooltip, useTooltipTrigger } from 'react-aria';
+import type { TooltipTriggerProps } from 'react-stately';
+import { useTooltipTriggerState } from 'react-stately';
+import { tooltipPositionVariants } from './Tooltip.css';
+export interface ITooltipProps extends Omit<TooltipTriggerProps, 'trigger'> {
+  children: ReactElement;
```

Comment:
Just double checking; we need to use `ReactElement` here as opposed to `ReactNode`?

Reply:
Yes, because we are applying a ref and props to the child element which means it should only wrap one element. This guards against the user doing something like this:
```
<Tooltip content="example">
  <Button />
  <Button />
  <Button />
</Tooltip>
```

We are expecting that the tooltip is applied to only one item